ghost/admin/app/templates/offers.hbs

```diff
@@ -92,9 +92,9 @@
                                     <span class="midgrey strike ml2">{{currency-symbol offer.finalCurrency}}{{gh-price-amount offer.originalPrice}}</span>
                                 {{/if}}
                             </LinkTo>
-                            <LinkTo @route="offer" @model={{offer}} class="gh-list-data">
+                            <a href="#" {{on "click" (fn this.offerMembers offer.id)}}  class="gh-list-data pointer">
```

Comment:
This can stay as a `<LinkTo>` so the behaviour is kept in the template and easily visible, query params can be set like this:

```
<LinkTo @route="members" @query={{hash filter=(concat "offer_redemptions:" offer.id)}} class="gh-list-data">
```